### ISSUE TITLE
WCAG 2.4.6 🟠 (Partially supports): Headings and Labels

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#5151]
**Level:** AA
**Principle:** Operable
**Result:** 🟠 Partially Supports

### Resources
[WCAG 2.2 Success Criterion](https://www.w3.org/WAI/WCAG22/Understanding/headings-and-labels.html)
> Headings and [labels](https://www.w3.org/WAI/WCAG22/Understanding/headings-and-labels#dfn-label) describe topic or purpose.

### Audit Checklist
- [x] Review previous Audit Result
- [x] Review work identified in previous audit and list open issues.
- [x] Check for any other relevant open issues.
- [x] Retest open issues, are these issues still present?
- [x] Test for any new issues that may have been introduced.
- [x] Determine compliance status, add this to issue name.
- [x] Add any new issues to the Accessibility project for triage.

### Previous audit
#4131
> ❌ Our internal page structures do not have a consistent heading levels. Not all our headings are descriptive.

### Automated Testing
| Axe Rule | Description | violations |
|--|--|--|
| empty-heading | Headings should not be empty | Press Homepage of OLH and Clean both have a single empty heading |
| heading-order | Heading levels should only increase by one | 5 issues found between all three themes |
| label | Form elements must have labels | None for clean, but OLH and Material have a handful |
| label-title-only | Form elements should have a visible label |  all pass |
| page-has-heading-one | Page should contain a level-one heading | Clean and OLH press homepage are missing H1 |

 ### Progress since last audit
- #4232 
- #4229

 ### Audit Notes
Progress has been made, particularly on creating consistent well-structured headings across all themes.  However there are a handful of places where failures have been found.

 ### Work arising
- #5245
